config/locales/views.en.yml

```diff
@@ -188,3 +188,41 @@ en:
       state:
         saving: Saving
         saved: Saved
+  layouts:
+    mobile_navbar:
+      button:
+        menu: Menu
+    sidebar:
+      image:
+        alt:
+          logo: CASA Logo
+      menu_item:
+        supervisors: Supervisors
+        volunteers: Volunteers
+        case_contacts: Case Contacts
+        emancipation_checklists: Emancipation Checklist(s)
+        admins: Admins
+      footer:
+        location: Location
+        role: Role
+      link:
+        edit_profile: Edit Profile
+        generate_court_reports: Generate Court Reports
+        generate_reports: Generate Reports
+        system_imports: System Imports
+        edit_organization: Edit Organization
+        report_site_issue: Report a site issue
+    application:
+      javascript_instrucion:
+        enable: Please enable javascript
+        require: This app requires javascript to work
+    login_header:
+      image:
+        logo:
+          alt: CASA Logo
+    footer:
+      not_logged_in:
+        built_by: Built by
```

Comment:
I wish that "Built by Ruby for Good" was all one line someday, but maybe that would be hard in the code- right now I think we have `<br>` instead of CSS

Reply:
the root key should be `footers` instead of `footer` :D I've just fixed it

Reply:
Yeah, I want that too .-.